### PR TITLE
Interface for getting range of row_ids

### PR DESCRIPTION
**Pull request by danpovey**

### CODE REVIEW

**Comment by danpovey** [comment]
This is something else I need implemented..

**Comment by qindazhu** [comment]
Will do

File: k2/csrc/array_ops.h [hunk 1]
```diff
@@ -378,6 +378,24 @@ Array1<T> Arange(ContextPtr c, T begin, T end, T inc = 1);
 void RowSplitsToRowIds(const Array1<int32_t> &row_splits,
                        Array1<int32_t> *row_ids);
 
+
+/*
+  This function returns *part* of a row_ids array (it's intended for situations
+  where computing the entire row_ids array would use too much memory).
+    @param [in] row_splits  Input row_splits vector, of dimension num_rows+1
+    @param [in] start_row   The first row that we want the row_id value for,
+                            with 0 <= start_row < row_splits.Back()
```
Comment (qindazhu) [inline k2/csrc/array_ops.h:387]: `0 <= start_row < num_rows` ?

Comment (qindazhu) [reply k2/csrc/array_ops.h:387]: if it's `0 <= start_row < row_splits.Back()`, Is `start_index` a better name?

Comment (danpovey) [reply k2/csrc/array_ops.h:387]: you're right, or start_element.  start_index is fine.

**Pull request closed without merging**